tests/gas_values.csv

```diff
@@ -0,0 +1,20 @@
+KEY GAS_COST GAS_LIMIT
+TRANSFER_0_MEDIATOR 60256 61000
+TRANSFER_1_MEDIATOR 91215 92000
+TRANSFER_2_MEDIATORS 122768 123000
+TRANSFER_3_MEDIATORS 153862 154000
+FIRST_TL_REQUEST 74470 75000
+SECOND_TL_REQUEST 39670 40000
+FIRST_TL 310533 341000
+UPDATE_TL 29341 59000
+REDUCE_TL_LIMITS 49449 50000
+CLOSE_TL_NO_TRANSFER 46103 93000
+CLOSE_TL_TRIANGULAR_TRANSFER_2_MEDIATORS 91374 182000
+CLOSE_TL_TRIANGULAR_TRANSFER_4_MEDIATORS 153773 244000
+CANCEL_TL_UPDATE 18830 38000
+DEPLOY_IDENTITY 1605565 -1
+DEPLOY_PROXIED_IDENTITY 283318 -1
+UNPROXIED_META_TRANSACTION_OVERHEAD 26729 -1
+PROXIED_META_TRANSACTION_OVERHEAD_OVER_NONPROXY_METATRANSACTION 1308 -1
+UNPROXIED_META_TRANSACTION_OVERHEAD_OVER_OWNED 21919 -1
+UNPROXIED_OWNED_TRANSACTION_OVERHEAD_OVER_REGULAR_TRANSACTION 4810 -1
```

Comment:
I don't get how / why we get -1 as a limit.

Reply:
I am not sure the gas limit is meaningless for overheads, since the clientlib needs to use a value for the overhead of using a meta-tx to the gas limit. But I guess we can make it easy by assuming there is no difference in between gas limit and gas usage for this and use the value of `UNPROXIED_META_TRANSACTION_OVERHEAD + PROXIED_META_TRANSACTION_OVERHEAD_OVER_NONPROXY_METATRANSACTION` which is about `28k` in the clientlib.

Reply:
Ok, I guess you are right, that it has a meaning, but I would at least not know how to test it properly. I realized that the same flow I do for the currency networks will not work. 
Also, we did not have them before, so if we feel the need to add them, we should just do it in another PR.